test/functional/apps/management/_test_huge_fields.js

```diff
@@ -23,7 +23,7 @@ export default function ({ getService, getPageObjects }) {
   const esArchiver = getService('esArchiver');
   const PageObjects = getPageObjects(['common', 'home', 'settings']);
 
-  describe('test large number of fields', function () {
+  describe('@skipcloud test large number of fields', function () {
```

Comment:
I can't think of a better way to do this though, aside from hard coding the test names in the cloud test runner, but I'd prefer if we put directives like this at the end of the test/suite name.